pages/dkp/konvoy/2.2/release-notes/index.md

```diff
@@ -54,6 +54,13 @@ The default value for flag `--with-aws-bootstrap-credentials` will be changing f
 
 #### New command to update Azure credentials in running clusters
 
+#### Default update strategy changed to "delete first" for Preprovisioned clusters
```

Comment:
Is this a known issue? work around? We probably need this in it's own section so it's not mistaken as a feature.

Reply:
I agree, it belongs in another section, because it's a bug fix. I didn't catch which section I was in.

Reply:
I added a "Fixes and Improvements" heading.